LanguageFeatures/Inline-classes/static_analysis_inline_class_A16_t01.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A compile-time error occurs if an inline class constructor
+/// includes a superinitializer
+///
+/// @description Checks that a compile-time error occurs if an inline class
+/// constructor includes a superinitializer
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=inline-class
+
+inline class V1 {
+  final int id;
+  V1(this.id);
+  V1.x(this.id);
+}
+
+inline class V2 implements V1 {
+  final int id = 0;
+  V2() : super();
```

Comment:
I think we should change the first `V1` constructor to accept an optional positional parameter. Otherwise we'll have an error no matter what, because it is an error to pass no actual arguments to the constructor named `V1`, and then we might not detect whether or not there is a bug in the check that makes an error to have `super()`, no matter how correct it looks relative to the constructors declared by `V1`.